### PR TITLE
T05 Completed

### PR DESCRIPTION
**Pull request by YoganathanM**

### CODE REVIEW

File: Assignment/Program.cs [hunk 1]
```diff
@@ -1,7 +1,17 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter number to be checked :");
+   if (int.TryParse (Console.ReadLine (), out int num)) {
+      int count = 0;
```
Comment (varalakshmi-imm) [inline Assignment/Program.cs:4]: Why do you need this count?

File: Assignment/Program.cs [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+   if (int.TryParse (Console.ReadLine (), out int num)) {
+      int count = 0;
+      for (int i = 2; i < num; i++) {
+         if (num % i != 0)
+            count++;
+         else {
+            Console.Write ($"{num} is not a prime number");
+            break;
+         }
+         if (count == num - 2)
```
Comment (varalakshmi-imm) [inline Assignment/Program.cs:12]: If it is not divisible by a number then continue and if it is divisible you break saying it is not a prime number.
I don't understand why you need this count == num - 2.

File: Assignment/Program.cs [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+            Console.Write ($"{num} is a prime number");
       }
-   }
+      break;
```
Comment (varalakshmi-imm) [inline Assignment/Program.cs:15]: Why do you need this break here?

**Review by varalakshmi-imm — CHANGES_REQUESTED** [review]
You don't have to push anything in main for any assignment only changes to be done in the particular branch.

File: Assignment/Program.cs (hunk 1 above)
Comment (YoganathanM) [reply Assignment/Program.cs]: Printing the output based on the count
Example: Given number is 11
Here i'm checking from 2 to 10 and taking the count
Then comparing the stored count with the expected count

File: Assignment/Program.cs (hunk 3 above)
Comment (YoganathanM) [reply Assignment/Program.cs:10]: I'm having a loop to handle invalid input
Example: T20 or $20
The loop will throw a message as an invalid number and this loop will continue until the user gives valid number
If valid input is validated then the loop will break

**Review by varalakshmi-imm — APPROVED** [review]

**Pull request closed without merging**